### PR TITLE
GOV-487 PACKAGES input

### PR DESCRIPTION
**Pull request by xeger**

### CODE REVIEW

**Comment by lopaka** [comment]
Metadata errors running `right_st rightscript validate install-packages.sh`

File: rll-examples/install-packages.sh [hunk 1]
```diff
@@ -2,16 +2,30 @@
 # ---
 # RightScript Name: SYS packages install
 # Description: |
-#   Installs packages required by RightScripts.
+#   Installs packages required by RightScripts, or extra packages.
 #   To handle naming variations, prefix package names with "yum:" or "apt:".
-# Inputs:
-# ...
```
Comment (lopaka) [inline rll-examples/install-packages.sh:8]: Still need this at the end to terminate metadata.

File: rll-examples/install-packages.sh [hunk 2]
```diff
@@ -2,16 +2,30 @@
...
+# Inputs: {}
```
Comment (lopaka) [inline rll-examples/install-packages.sh]: `{}` needs to be removed since it is not to be empty.

**Comment by xeger** [comment]
Thanks; fixed in amended commit.

**Comment by lopaka** [comment]
LGTM

**Pull request merged**